src/components/Popup.js

```diff
@@ -7,6 +7,8 @@ export default class Popup {
     this._modalContainer =
       this._popupElement.querySelector(".js-modalContainer");
 
+    this._submitButton = this._popupElement.querySelector(".modal__button");
```

Comment:
Needs correction: the popup class should not know anything about the submit button as it renders only the general modal